spirit/category/models.py

```diff
@@ -27,6 +27,8 @@ class Category(models.Model):
 
     # topic_count = models.PositiveIntegerField(_("topic count"), default=0)
 
+    color = models.CharField(_("color"), max_length=7, null=True, blank=True)
```

Comment:
Oops, I actually never pass `null=True`, I should have checked it :cold_sweat: . In fact [It is a good practice not to set null to true for charfields](https://docs.djangoproject.com/es/1.9/ref/models/fields/#django.db.models.Field.null).

Can you reverse it back? so sorry!